internal/pkg/describe/app.go

```diff
@@ -23,13 +23,14 @@ import (
 
 // App contains serialized parameters for an application.
 type App struct {
-	Name      string                   `json:"name"`
-	Version   string                   `json:"version"`
-	URI       string                   `json:"uri"`
-	Envs      []*config.Environment    `json:"environments"`
-	Services  []*config.Workload       `json:"services"`
-	Jobs      []*config.Workload       `json:"jobs"`
-	Pipelines []*codepipeline.Pipeline `json:"pipelines"`
+	Name         string                   `json:"name"`
+	Version      string                   `json:"version"`
+	URI          string                   `json:"uri"`
+	Envs         []*config.Environment    `json:"environments"`
+	Services     []*config.Workload       `json:"services"`
+	Jobs         []*config.Workload       `json:"jobs"`
+	Pipelines    []*codepipeline.Pipeline `json:"pipelines"`
+	WorkloadEnvs map[string][]string      `json:"-"`
```

Comment:
Hmm i think ideally the `json` format should show more info than the human output.

Reply:
I feel like we probably don't want to include the deployed env information now. It's a nice thing to show in human string, but I feel like we want to leave the door open to take it away.

Reply:
Ok sounds good. Maybe we can add it in the future whenever it is either requested by customers or we need it from the other commands.